swagger/model_builder.go

```diff
@@ -65,8 +65,12 @@ func (b modelBuilder) addModel(st reflect.Type, nameOverride string) *Model {
 
 	// check for slice or array
 	if st.Kind() == reflect.Slice || st.Kind() == reflect.Array {
-		b.addModel(st.Elem(), "")
-		return &sm
+		if st.Elem().Kind() == reflect.Uint8 {
+			return nil
```

Comment:
is this returning nil because we're representing it as a "string", which is already added as a model? If so, I would put this case up above, right after the isPrimitiveType check, with a really good comment